huaweicloud/common/ecs_instance_keypair.go

```diff
@@ -1,4 +1,4 @@
-package huaweicloud
+package common
 
```

Comment:
those func is only used by esc_instance. 
it‘s better to move this two file to package ecs

Reply:
it will be used by CCE node later